features/antivirus/antivirus.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Antivirus scanning
+
+Scenario: Uploading an infected file to a bucket should generate a warning email
+  Given I have a randomized file continaing the eicar test signature
+  When I upload that file to the documents bucket under the key 'functional_tests/bad.pdf'
+  Then I receive a notification regarding that file within 4 minutes
```

Comment:
Is the plan to have a non-infected file upload scenario as well, where no email is sent? 

I'm not sure about having this as a smoke test, unless we can clean up the 144 random files per day being uploaded to our documents buckets for each stage?

Reply:
Agree with @katstevens on both points

Reply:
I thought about this when I was going to sleep the other night actually, and I concluded "look, if it were sending spurious emails even about clean uploads, we'd damn well know about it because we'd be getting spammed with them".

I just feel a bit nervous about leaving production un-smoke-tested, because a whole *bunch* of infrastructure things need to be in place and applied correctly for the whole loop to be working. Without this check we'd never realize if our SNS subscription got accidentally removed by a terraform apply or altered, re-added and never managed to complete the SubscriptionConfirmation process. Anything like that.

As for the 144 files, the severity of that is slightly reduced by them all being under the same key, but yeah, running so frequently would feel a bit silly. This is why I think there's a need for some kind of reduced-frequency smoke test...?

One of the other motivations I had for having the smoke tests performing this is to keep the avapi's logs full. Without this constant trickle, we're unable to do the HG "no logs" alert.

Reply:
Good point about the HG alerts. I'm happy for a new `@smoulder` tag and job to exist - once per hour seems manageable?